src/app/branded/step-1/branded-checkout-step-1.tpl.html

```diff
@@ -44,6 +44,40 @@ <h3 class="panel-name" translate>{{'PAYMENT'}}</h3>
       </checkout-step-2>
     </div>
   </div>
+  <div class="panel" ng-if="$ctrl.premiumCode">
+    <div class="panel-body">
+      <h3 class="panel-name" translate>{{'CHOOSE_RESOURCE'}}</h3>
+      <div class="row">
+        <div class="col-sm-3">
+          <div class="row">
+            <label class="radio-inline">
+              <input
+                type="radio"
+                name="premiumSelect"
+                ng-model="$ctrl.premiumSelected"
+                ng-value="true"
+                ng-change="$ctrl.onSelectPremiumOption()">
+              {{$ctrl.premiumName}}
+            </label>
+          </div>
+          <div class="row">
+            <label class="radio-inline">
+              <input
+                type="radio"
+                name="premiumSelect"
+                ng-model="$ctrl.premiumSelected"
+                ng-value="false"
+                ng-change="$ctrl.onSelectPremiumOption()">
+              <translate>{{'NO_THANK_YOU'}}</translate>
+            </label>
+          </div>
+        </div>
+        <div class="col-sm-4">
+          <img class="premium-thumbnail" ng-src="{{$ctrl.premiumImageUrl}}" />
```

Comment:
We should make this div conditional on the existence of the premium image url since it is optional.